systems/x86_64-linux/boba-social/default.nix

```diff
@@ -8,7 +8,24 @@
   services.openssh.enable = true;
   # Might need to run `sudo tailscale up --ssh=false` if SSH hangs on update
   services.tailscale.enable = true;
-  services.bobaboard.enable = true;
+  networking.firewall.allowedTCPPorts = [
+    22
+  ];
+
+  services.bobaboard = {
+    enable = true;
+    database = {
+      user = "bobaboard";
+      host = "db-postgresql-nyc3-75986-do-user-12927016-0.b.db.ondigitalocean.com";
+      local = false;
+      port = 25060;
+      name = "bobadb";
+      passwordFile = "/var/lib/bobaboard/db-password";
+      sslRootCertPath = "/var/lib/bobaboard/db-ca";
```

Comment:
This doesn't look right to me, where did you get this certificate from?

Reply:
Here: https://docs.digitalocean.com/products/databases/postgresql/how-to/connect/

It's a digital ocean managed db thing, I also was confused at first.

Reply:
Hmm, interesting. In that case this is alright, it'll just make it more difficult to reproduce since there are additional manual steps. Maybe as time goes on we can look at self hosting the DB too.

Reply:
That's the idea, but first I want to get it to work with a managed DB :)

We might want to have a version with a local DB for stuff like testing environment, even before I'm comfortable with not using a managed service for it. But not right now anyway!